tests/test_project_generation.py

```diff
@@ -0,0 +1,31 @@
+import pybamm_cookiecutter as m
+import pytest
```

Comment:
```suggestion
import pytest
from pytest_cookies.plugin import Cookies
```

Related to the other typing comment.